sites/public/pages/finder.tsx

```diff
@@ -0,0 +1,211 @@
+import {
+  encodeToFrontendFilterString,
+  FrontendListingFilterStateKeys,
+  ListingFilterState,
+} from "@bloom-housing/shared-helpers"
+import {
+  AppearanceStyleType,
+  Button,
+  Field,
+  Form,
+  FormCard,
+  ProgressNav,
+  StepHeader,
+  t,
+} from "@bloom-housing/ui-components"
+import axios from "axios"
+import router from "next/router"
+
+import React, { useEffect, useState } from "react"
+import { useForm } from "react-hook-form"
+import Layout from "../layouts/application"
+
+const getTranslationString = (str: string) => {
+  if (str === "studio") {
+    return "studioPlus"
+  } else if (str === "oneBdrm") {
+    return "onePlus"
+  } else if (str === "twoBdrm") {
+    return "twoPlus"
+  } else if (str === "threeBdrm") {
+    return "threePlus"
+  } else if (str === "fourBdrm") {
+    return "fourPlus"
+  }
+}
+
+interface finderField {
+  label: string
+  translation: string
+  selected: boolean
+}
+
+interface finderQuestion {
+  fieldGroupName: string
+  fields: finderField[]
+}
+
+interface finderForm {
+  [key: number]: finderQuestion
+}
+
+const Finder = () => {
+  const [currentStep, setCurrentStep] = useState<number>(1)
+  const [completedSteps, setCompletedSteps] = useState<number>(0)
+  const [activeQuestion, setActiveQuestion] = useState<finderQuestion>(null)
+  const formData: finderForm = {}
+
+  const onSubmit = (data: ListingFilterState) => {
+    void router.push(`/listings/filtered?page=${1}&limit=${8}${encodeToFrontendFilterString(data)}`)
+  }
+  useEffect(() => {
+    const getAndSetOptions = async () => {
+      try {
+        const response = await axios.get(`${process.env.backendApiBase}/listings/meta`)
+        // console.log(response.data)
+        if (response.data) {
+          if (response.data.unitTypes) {
+            const bedroomFields = response.data.unitTypes.map((elem) => ({
+              label: elem.name,
+              translation: getTranslationString(elem.name),
+              selected: false,
+            }))
+            formData[1] = { fieldGroupName: "bedRoomSize", fields: bedroomFields }
+          }
+          setActiveQuestion(formData[1])
+        }
+      } catch (e) {
+        console.error(e)
+      }
+    }
+    void getAndSetOptions()
+  }, [])
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register, handleSubmit, getValues } = useForm()
+
+  const ProgressHeader = (currentStep: number, completedSteps: number) => {
+    return (
+      <div className="flex flex-col w-full">
+        <div className="flex flex-row justify-between">
+          <div className="md:text-xl capitalize font-bold">
+            {t("listingFilters.buttonTitleExtended")}
```

Comment:
In mobile view the header and stepHeader doesn't have any padding between it and the site header
![image](https://user-images.githubusercontent.com/42942267/191530965-88ef2b19-58da-40aa-a4af-f6f6a0e0951a.png)